edge/pkg/edged/edged.go

```diff
@@ -913,13 +920,14 @@ func (e *edged) consumePodAddition(namespacedName *types.NamespacedName) error {
 	}
 
 	podUID := pod.GetUID()
-	curPodStatus, err := e.podCache.GetNewerThan(podUID, e.podLastSyncTime[podUID])
+	curPodStatus, err := e.podCache.GetNewerThan(podUID, e.lastSyncTime[podUID])
 	if err != nil {
 		klog.Errorf("pod %s cache newer failed: %v", podName, err)
 		return err
 	}
 	result := e.containerRuntime.SyncPod(pod, curPodStatus, secrets, e.podAdditionBackoff)
-	e.podLastSyncTime[podUID] = time.Now()
+	e.lastSyncTime[podUID] = time.Now()
```

Comment:
Remove this line

Reply:
> Remove this line

Here should call set function. I make a mistake.